### PR TITLE
[traceback] Expose print_list method in traceback for python3.

### PR DESCRIPTION
**Pull request by lic9**
Add print_list to traceback based on https://github.com/python/cpython/blob/master/Lib/traceback.py#L19.

### CODE REVIEW

File: stdlib/2and3/traceback.pyi [hunk 1]
```diff
@@ -36,6 +36,7 @@ if sys.version_info >= (3, 5):
     def extract_stack(f: Optional[FrameType] = ...,
                       limit: Optional[int] = ...) -> StackSummary: ...
     def format_list(extracted_list: List[FrameSummary]) -> List[str]: ...
+    def print_list(extracted_list: List[FrameSummary], file: Optional[IO[str]] = ...) -> None: ...
```
Comment (srittau) [inline stdlib/2and3/traceback.pyi]: 1. This should be marked with `# undocumented`.
2. While `StackSummary.from_list()` is also annotated as expecting a list, expecting an `Iterable[FrameSummary]` should suffice.
3. `print()` is annotated as expecting an `IO[str]`. We are trying to move away from using `IO` for argument types. Could you defined a protocol like this and use it for the file argument:
```python
class _Writable(Protocol):
    def write(self, s: str) -> Any:
        pass
```

I will send a separate PR for `print()`.

Comment (lic9) [reply stdlib/2and3/traceback.pyi]: Will do. Thanks for the commends and suggestions!

Comment (srittau) [reply stdlib/2and3/traceback.pyi]: For reference, the print PR is #2848.

Comment (lic9) [reply stdlib/2and3/traceback.pyi]: Looks like all tests have finished but status is still pending...

**Comment by srittau** [comment]
I don't know why it's still showing as pending, but since all tests are green, I'm meeting.

**Pull request merged**